tests/00.databaseConnectivity.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { test, expect } from '@playwright/test';
+import { runQuery } from '../utils/oracle';
+
+test('verify Oracle DB connectivity', async () => {
+  const result = await runQuery(`SELECT 'd[-_-]b' AS test FROM dual`);
```

Comment:
although this obviously looks funnier it would probably be sensible to just do `SELECT 1` - definitely took me a few seconds of looking to work out what was meant to be going on here, will probably make less sense in 5 years' time. Or add a comment I guess.

Reply:
lol :D fair enough I've added a comment. I don't see us keeping this test for very long. There will be no need to verify DB connectivity when we start using the DB for actual data provisioning and assertions.